test/NuGet.Core.FuncTests/Dotnet.Integration.Test/PackCommandTests.cs

```diff
@@ -0,0 +1,392 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using NuGet.Frameworks;
+using NuGet.Packaging;
+using NuGet.Test.Utility;
+using NuGet.Versioning;
+using NuGet.XPlat.FuncTest;
+using Xunit;
+
+
+namespace NuGet.Commands.FuncTest
+{
+    [Collection("Dotnet Integration Tests")]
```

Comment:
In this case, DotnetIntegrationCollection is what drives the fixture : https://github.com/NuGet/NuGet.Client/blob/d462f917c769286c697ddb82bfd95d0e53903b9d/test/NuGet.Core.FuncTests/Dotnet.Integration.Test/DotnetIntegrationCollection.cs